skeletor/common/css.config.js

```diff
@@ -1,5 +1,13 @@
 const path = require('path');
```

Comment:
[suggestion] postcss-url will be expecting an object passed in. I would add an empty object and some documentation that explains this.
Something like the following
```suggestion
/**
 * Sets the paths and plugins to be used for all of the CSS tasks run by Skeletor
 *
 * @const assetsMap {Object} mapping of local URLs and remote URLs or empty Object
 * @note assetsMap needs to be an Object with the original url as the key and the new url as the value
 * You can also create a JSON file elsewhere - such as assets.json - and require it instead of listing them here
 * @example
 * 	assetsMap = {
 * 	    "images/my_image.png": "https://www.deg.com/image/on/the/server.png",
 * 	    "images/my_second_image.png": "https://www.deg.com/second/image/on/the/server.png"
 * 	    ...
 * 	}
 * 	*/
const path = require('path');
const assetsMap = {};
```

Reply:
Done

Reply:
Done